.golangci.example.yml

```diff
@@ -186,6 +186,10 @@ linters-settings:
         paramsOnly: true
       rangeValCopy:
         sizeThreshold: 32
+  godox:
+    keywords:
```

Comment:
please, leave a comment here that default keywords are `TODO`, `BUG`, `FIXME` and here we can specify which keywords we need and they will override default keywords.

Reply:
Added.